tools/docker/Dockerfile

```diff
@@ -7,17 +7,22 @@ ENV DEBIAN_FRONTEND=noninteractive \
 # General requirements not in the base image
 RUN apt-get -qqy update \
   && apt-get -qqy install \
+    bridge-utils \
     bzip2 \
     ca-certificates \
     dbus-x11 \
     earlyoom \
     fluxbox \
     gdebi \
     git \
+    libvirt-daemon-system \
+    libvirt-clients \
     locales \
+    openjdk-8-jre-headless \
```

Comment:
Q: does the SDK require JDK 8? Can we use the generic/versionless package to be more future-proof?

Reply:
That would be nice, wouldn't it. Unfortuantely until I used this older version I was getting some mysterious failures, so I think it's really required.